phase2/ignition/vanilla/kubelet.service

```diff
@@ -6,6 +6,7 @@ Documentation=https://github.com/kubernetes/kubernetes
 ExecStartPre=/bin/mkdir -p /var/lib/kubelet
 ExecStartPre=/bin/mount --bind /var/lib/kubelet /var/lib/kubelet
 ExecStartPre=/bin/mount --make-shared /var/lib/kubelet
+ExecStartPre=/bin/mkdir -p /var/log/containers
```

Comment:
Gotcha.  I was unaware of that.  I added and removed that line probably about half a dozen times during my troubleshooting.